src/appengine/libs/issue_management/issue_filer.py

```diff
@@ -387,6 +387,13 @@ def file_issue(testcase,
   else:
     issue.status = properties.status
 
+  fuzzer_metadata = testcase.get_metadata('issue_metadata')
+  if fuzzer_metadata and 'issue_assignee_override' in fuzzer_metadata:
+    issue.status = policy.status('assigned')
+    issue.assignee = fuzzer_metadata['issue_assignee_override']
+    logs.log(
+        'Fuzzer is under staging. Assigned testcase to %s' % issue.assignee)
```

Comment:
nit: Can you make this message a bit more generic? The "staging" bit is a bit of an implementation detail for Android.

e.g. "Testcase has assignee metadata (%s)".